mappings/net/minecraft/world/gen/Spawner.mapping

```diff
@@ -1,2 +1,13 @@
 CLASS net/minecraft/class_5304 net/minecraft/world/gen/Spawner
+	COMMENT Handles spawning entities in a world.
+	COMMENT
+	COMMENT <p>A spawner is typically used to spawn entities within a special context, such as cats in a village or wandering traders.
 	METHOD method_6445 spawn (Lnet/minecraft/class_3218;ZZ)I
+		COMMENT Spawns entities into a world.
+		COMMENT
+		COMMENT @return the amount of entities that were spawned. 0 if no entities were spawned.
```

Comment:
```suggestion
		COMMENT @return the count of entities spawned
```

Reply:
I'd say `the number of entities spawned` then. `count of entities` really doesn't sound right